website/docs/r/nlb_service.html.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: "exoscale"
+page_title: "Exoscale: exoscale_nlb_service"
+sidebar_current: "docs-exoscale-nlb-service"
+description: |-
+  Provides an Exoscale Network Load Balancer Service resource.
```

Comment:
```suggestion
  Provides an Exoscale Network Load Balancer service resource.
```

Since it's a noun in this context, *service* should not be capitalized (applies everywhere).

Reply:
4bd3625